facebook_scraper.py

```diff
@@ -259,3 +260,23 @@ def _login_user(email, password):
     _session.post(_base_url + login_action, data={'email': email, 'pass': password})
     if 'c_user' not in _session.cookies:
         warnings.warn('login unsuccessful')
+
+def get_posts_as_csv(account, pages=10, timeout=5, sleep=0, credentials=None):
+    """
+
+    :param pages:
+    :param timeout:
+    :param sleep:
+    :param credentials:
+    :return:
+    """
+    list_of_posts = []
+    for current_post in get_posts(account, pages, timeout, sleep, credentials):
+        list_of_posts.append(current_post)
+    keys = list_of_posts[0].keys()
```

Comment:
Check that list_of_posts is not empty first. The group could have no posts, or the credentials could be invalid.